### DISCUSSION TITLE
Multidimensional Time Series Clustering

### DISCUSSION THREAD

**Discussion by TGNYC** [tslearn-team/tslearn#409] [Q&A]
I could not figure it out from the documentation, but I am attempting to cluster time series data that has equal time intervals but two features per time event. I am unsure about how to import this data into `tslearn` and the TimeSeriesKMeans did not seem to work. Any help would be appreciated! Best.

**Comment by GillesVandewiele** [comment]
That would be a multi-variate time series. In tslearn, your time series can be modeled as a 3D `(n_ts, ts_length, n_dim)` tensor (see [our documentation](https://tslearn.readthedocs.io/en/stable/gettingstarted.html#time-series-format)). TimeSeriesKMeans accepts multi-variate input according to the documentation (https://tslearn.readthedocs.io/en/stable/gen_modules/clustering/tslearn.clustering.TimeSeriesKMeans.html#tslearn.clustering.TimeSeriesKMeans.fit)
